Code/GraphMol/ForceFieldHelpers/CrystalFF/TorsionPreferences.cpp

```diff
@@ -160,6 +160,24 @@ void getExperimentalTorsions(const RDKit::ROMol &mol, CrystalFFDetails &details,
     throw ValueErrorException("molecule has no atoms");
   }
 
+  class VerboseSaver {
+    bool d_originallyEnabled = false;
+
+   public:
+    VerboseSaver(bool newState) {
+      if (rdInfoLog) {
+        d_originallyEnabled = rdInfoLog->df_enabled;
+        rdInfoLog->df_enabled = newState;
+      }
+    }
+    ~VerboseSaver() {
+      if (rdInfoLog) {
+        rdInfoLog->df_enabled = d_originallyEnabled;
+      }
+    }
+  };
```

Comment:
Should we put this into RDLog ?

Reply:
We could, but this is hopefully something we won't be using a lot. 
The `verbose` argument here isn't really needed if the info log is disabled by default: in future situations like this someone who wants verbose output from a function could just enable the info log before calling the function.

Reply:
I could actually see a more generic class in RDLog actually.  BlockLogs blocks everything, but you could have a temporary state setter

```
RDLog::LogState logstate(RD_ENABLE_INFO | RD_ENABLE_VERBOSE | RD_DISABLE_DEBUG );
```

Which would set temporary log states Then BlockLogs would subclass from this.

Reply:
@bp-kelley nice idea! I added something along these lines.